src/modules/core/components/Tag/Tag.tsx

```diff
@@ -14,6 +14,7 @@ export interface Appearance {
     | 'danger'
     | 'pink'
     | 'blue'
+    | 'banned'
```

Comment:
What's the point of this, if you've added a separate `core` component for the banned tag in the end ?

Reply:
I'm wrapping Tag with the banned theme and the goblin emoji on the new component

Would you prefer me to not use Tag here and move the banned theme to the BannedTag component css's instead?

```
const BannedTag = ({ children, text, textValues, ...rest}: Props) => {
  return (
    <Tag appearance={{ fontSize: 'small', theme: 'banned', margin: 'none' }}>
      <Icon
        title="banned"
        name="emoji-goblin"
        appearance={{ size: 'normal' }}
        className={styles.icon}
      />
      <div className={styles.textSyles}>
              {text ? (
        <>
          {typeof text === 'string' ? (
            text
          ) : (
            <FormattedMessage {...text} values={textValues} />
          )}
        </>
      ) : (
        children
      )}
      </div>
    </Tag>
  );
};
```

Reply:
Moved all code to new component, no Tag modifications